boot/bootutil/include/bootutil/image.h

```diff
@@ -82,16 +87,24 @@ struct image_version {
     uint32_t iv_build_num;
 };
 
+struct image_dependency {
+    uint32_t image_id;                      /* Image index (from 0) */
+    struct image_version image_min_version; /* Indicates at minimum which
+                                             * version of firmware must be
+                                             * available to satisfy compliance
+                                             */
+};
+
 /** Image header.  All fields are in little endian byte order. */
 struct image_header {
     uint32_t ih_magic;
     uint32_t ih_load_addr;
-    uint16_t ih_hdr_size; /* Size of image header (bytes). */
-    uint16_t _pad1;
-    uint32_t ih_img_size; /* Does not include header. */
-    uint32_t ih_flags;    /* IMAGE_F_[...]. */
+    uint16_t ih_hdr_size;           /* Size of image header (bytes). */
+    uint16_t ih_protect_tlv_size;   /* Size of protected TLV area (bytes). */
```

Comment:
in place of _pad1

Reply:
`ih_protect_tlv_size` field should be declared only if (BOOT_IMAGE_NUMBER > 1) is met.

Reply:
I don't think it is a problem to just change the name since `_pad1` was not used before anyway.

Reply:
I see, right